kolibri_instant_schools_plugin/auth/api.py

```diff
@@ -101,15 +101,12 @@ def retrieve(self, request, pk=None):
 
 class FacilityUserProfileViewset(FacilityUserViewSet):
 
-    def perform_update(self, serializer):
+    def set_password_if_needed(self, instance, serializer):
```

Comment:
On FacilityUserViewset, the args list is just `(self, serializer)`. Will this override implementation still work?

Reply:
In kolibri's FacilityUserViewset, all calls to this method are `set_password_if_needed(serializer)`

Reply:
Ah, shoot, I had some final changes on Kolibri that didn't get pushed -- pushing those now.